packages/flutter_test/lib/src/goldens.dart

```diff
@@ -45,6 +45,16 @@ abstract class GoldenFileComparator {
   /// The method by which [golden] is located and by which its bytes are written
   /// is left up to the implementation class.
   Future<void> update(Uri golden, Uint8List imageBytes);
+
+  /// Updates the uri [key] of the golden file to incorporate any [version]
```

Comment:
"Updates" sounds like it mutates the given Uri object. Doesn't it return a new Uri object?